README.md

```diff
@@ -323,12 +323,13 @@ x-okapi-token: [JWT_TOKEN]
 
 {
   "recreateIndex": true,
-  "resourceName": "instance"
+  "resourceName": "authority"
 }
 ```
 
-* `resourceName` parameter is optional and equal to `instance` by default. Possible values: `instance`, `authority`, `location`,
```

Comment:
Does it make sense to make the parameter required without default? 
For me it's strange to have some default for it, what do you think?

Reply:
Sound good to make it required